backend/server/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -689,7 +689,7 @@
                 <constraints primaryKey="true" nullable="false"/>
             </column>
             <column name="migration_name" type="varchar(255)"/>
-            <column name="status" type="tinyint(1)"/>
```

Comment:
since we're not using liquibase any more...is this change relevant?

Reply:
Reverted